### PR TITLE
Integrated linter in repo

### PR DESCRIPTION
**Pull request by warku123**
There are still some error if add `pylint` linter, which means there maybe still a few python grammar fault.

Now I remove the pylint linter, so the fix can run automatically.

You can see the pylint error here:
https://github.com/warku123/hypervisor-builder/actions/runs/4604288730/jobs/8135073837

### CODE REVIEW

**Comment by warku123** [comment]
Try to set up python environment, but still failed.

Can see the error info here:
https://github.com/warku123/hypervisor-builder/actions/runs/4624824148/jobs/8180027329

Commit: https://github.com/VirtwhoQE/hypervisor-builder/pull/28/commits/ab4fba144c9266570ad1dc3f98789612d94fce3b

**Review by hkx303 — APPROVED** [review]
ACK, good to see the python-black code in this project, pending a small comment

File: .mega-linter.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#   - REPOSITORY
+SHOW_ELAPSED_TIME: true
+FILEIO_REPORTER: false
+# DISABLE_ERRORS: true # Uncomment if you want MegaLinter to detect errors but not block CI to pass
+
+PRE_COMMANDS:
+  - command: pip --version && python -V
+  - command: pip install -r requirements.txt
+    cwd: "workspace"   # Will be run at the root of the workspace (usually your repository root)
+  - command: pip install urllib3
```
Comment (hkx303) [inline .mega-linter.yml]: still need the new line at of the file here

Comment (warku123) [reply .mega-linter.yml]: Fixed

**Review by hkx303 — APPROVED** [review]
ACK

**Pull request merged**